tools/printer-lg-png.h

```diff
@@ -1845,6 +1845,6 @@ static const unsigned char printer_lg_png[] =
   0x54,0xbd,0x25,0xba,0x49,0x71,0x9e,0x59,0xba,0x4d,0x53,0x1f,0xe6,0xed,0x4b,0xb9,
   0xde,0x17,0x45,0xf8,0x6b,0x81,0xbf,0xa8,0x8c,0x3b,0xff,0xff,0x00,0xa9,0xd9,0x66,
   0x33,0x79,0x3c,0x5d,0x0d,0x00,0x00,0x00,0x00,0x49,0x45,0x4e,0x44,0xae,0x42,0x60,
-  0x82,
+  0x82
```

Comment:
These are auto-generated and will get changed whenever the source PNG image is changed.